### ISSUE TITLE
Missing Pokemon every so often

### ISSUE THREAD

**Issue by waltwhitman6969** [kvangent/PokeAlarm#69]
For some reason I have noticed the alarm doesn't trigger a notifcation for some Pokemon. In the past two days. PokemonGoMap has picked up a Pinsir and a Rhydon near me but PokeAlarm had no notifcation of either.

The cmd prompt that I have running for PokeAlarm didn't see it at all.

I made sure that these two Pokemon are not marked as False in alarms.json and I do not know if there are any others at the moment, but those two are the ones that have been missed.

Any one else have this issue or similar?

**Comment by kvangent** [comment]
Issue was not in the proper template.

**Comment by waltwhitman6969** [comment]
You just answered another persons issue earlier without the proper template. Do you want me to describe the issue another way?

**Comment by kvangent** [comment]
@waltwhitman6969 I just recently added the issue template, so I gave an hour or so before I started enforcing it. I'm wasting a ton of time troubleshooting individual issues instead of adding improvements and actually fixing real bugs, so this is an attempt to streamline some.

If you checked the Alarm output and didn't see the name of the Pokemon (when it rejects a Pokemon it'll always say why) then it was because it didn't get the webhook. This could be that PokemonGo-Map experienced an error and didn't finish sending the webhook or something.
